CHANGES.txt

```diff
@@ -2,7 +2,7 @@
 
 * Transition from the ocurl package name to curl.
   (Nicolás Ojeda Bär, ygrek)
-* Split curl_lwt into its own package. The new library name is curl.lwt.
```

Comment:
Do we agree that `curl.lwt` was a mistake, it should have been `curl_lwt`?

Reply:
yes, I agree that it was a typo.